features/partners.feature

```diff
@@ -10,38 +10,121 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
+      | naam      | waarde |
+      | voornamen | Merel  |
 
     Scenario: Actueel partnerschap
-      Gegeven de persoon met burgerservicenummer 999993380 heeft de volgende huwelijken/partnerschappen in de registratie
-        | Categorie | soort verbintenis (15.10) | Voornamen (02.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | Onjuist (84.10) |
-        | 5         | P                         | Reindert          | 20040506              |                          |                 |
-      Als de persoon met burgerservicenummer 999993380 wordt geraadpleegd met fields=partners
-      Dan bevat het antwoord een partner met naam.voornamen "Reindert"
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999993380 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992935 |
+      En de partner heeft de volgende naam gegevens
+      | naam              | waarde   |
+      | voornamen (02.10) | Reindert |
+      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam                      | waarde   |
+      | datum aangaan (06.10)     | 20040506 |
+      | soort verbintenis (15.10) | P        |
+      | indicatie onjuist (84.10) |          |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993380                       |
+      | fields              | partners.naam.voornamen         |
+      Dan heeft de persoon met burgerservicenummer '999993380' alleen 'partners' met de volgende gegevens
+      | burgerservicenummer |
+      | 999992935           |
+      En heeft de partner met burgerservicenummer '999992935' de volgende 'naam' gegevens
+      | naam      | waarde   |
+      | voornamen | Reindert |
 
     Scenario: Huwelijk/partnerschap is ontbonden
-      Gegeven de persoon met burgerservicenummer 999992806 heeft de volgende huwelijken/partnerschappen in de registratie
-        | Categorie | Voornamen (02.10) | Datum aangaan (06.10) | Datum ontbinding (07.10) | Onjuist (84.10) |
-        | 5         | Osama             |                       | 20011109                 |                 |
-        | 55        | Osama             | 20000115              |                          |                 |
-      Als de persoon met burgerservicenummer 999992806 wordt geraadpleegd met fields=partners
-      Dan bevat het antwoord GEEN partners
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992806 |
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
```

Comment:
In de actuele categorie 5 partners zit -als het goed is ingevoerd- bij een ontbonden relatie alleen groep 7 (ontbinding): geen groep 6 (aangaan) en geen groep 2 (naam) en geen groep 1 (burgerservicenummer). Het burgerservicenummer, naam en aangaanHuwelijkPartnerschap zitten dan in historische categorie 55. Voor het illustreren van deze rule is dat allemaal niet zo relevant (alleen voor partnerhistorie), want alleen het voorkomen van ontbinding huwelijk partnerschap (07.10) is hier relevant. Ik zou dat hier weglaten

Reply:
wel is hier relevant dat er exact 1 partner is. Anders is het alsnog mogelijk dat de validatie (GEEN 'partners') niet klopt. je zit dan in de situatie van het volgende scenario (ontbonden plus actueel)